husky/app/src/main/java/com/keylesspalace/tusky/ViewMediaActivity.kt

```diff
@@ -458,6 +462,15 @@ class ViewMediaActivity : BaseActivity(), ViewImageFragment.PhotoActionsListener
             ).show()
         }
     }
+
+    // Prevent this activity from dimming or sleeping the screen if, and only if, it is playing video or audio
+    private fun adjustScreenWakefulness() {
+        if (attachments!![binding.viewPager.currentItem].attachment.type == Attachment.Type.IMAGE) {
```

Comment:
This is only for images, not video and audio too. The comment is misleading. Either add those types too, or change the comment, please.

Reply:
The comment indicates the intention, of keeping the screen awake for video and audio. The conditional on this line is to clear the Keep Screen On flag for static images, with the else clause used to set the flag the other types of attachments (gifv, video, audio, unknown).
I copied the code from Tusky, but if desired, I could invert the conditional so that the flag is set in `if` instead of `else`.